.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,21 @@
+name: Codespell
+
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+
+on: pull_request
+
+jobs:
+  codespell:
+    name: Codespell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Codespell
+        uses: codespell-project/actions-codespell@master
```

Comment:
Should we pin it to a dedicated version?

Reply:
As far as I see upstream has no releases. Would you like me to pin it to specific commit?